src/redux/categories/categories.js

```diff
@@ -0,0 +1,24 @@
+// Actions
+export const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
+
+// initial state
+const categories = [];
+
+// Action Creators
+export const checkStatus = () => ({
+  type: CHECK_STATUS,
+});
+
+// Reducer
+const categoryReducer = (state = categories, action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return [
+        ...state,
+      ];
```

Comment:
- [ ] Kindly make sure when the action CHECK_STATUS the function returns a string "Under construction".